### DISCUSSION TITLE
Mouse over text of the full job name in the jobs list

### DISCUSSION THREAD

**Discussion by lczech** [community/community#41364] [Actions]
### Select Topic Area

Product Feedback

### Body

In the jobs list on the left of a workflow run, the job names created from a matrix contain all params of the matrix, making the names potentially longer than the width of that column:

![image](https://user-images.githubusercontent.com/4741887/206876073-6205b8fd-a4c9-44be-a579-6e5b79e4196c.png)

It would help if those texts had a mouse over that contained the full name of the job.

I'm not sure that this is the right place for such suggestions; if not, please let me know where to post this.

**Comment by fmigneault** [comment]
Alternatively, the list could also be made wider or adjust its width more dynamically to available space. 
My workflows are never taking even half of the screen, even with everything expanded, but the side job list and items in the graph always insist on clipping the name for no reason.
![image](https://user-images.githubusercontent.com/19194484/222337537-f47b956e-06e6-4a6c-9f4a-a594061b3a88.png)

Yet another alternative, allow the names to extend over multiple lines.

Both of those can be easily fixed with some quick CSS changes, but it is annoying GitHub doesn't do it automatically by itself.
